web/components/portfolio/portfolio-value-section.tsx

```diff
@@ -0,0 +1,59 @@
+import { PortfolioMetrics } from 'common/user'
+import { formatMoney } from 'common/util/format'
+import { last } from 'lodash'
+import { memo, useState } from 'react'
+import { Period } from 'web/lib/firebase/users'
+import { Col } from '../layout/col'
+import { Row } from '../layout/row'
+import { PortfolioValueGraph } from './portfolio-value-graph'
+
+export const PortfolioValueSection = memo(
+  function PortfolioValueSection(props: {
+    portfolioHistory: PortfolioMetrics[]
+  }) {
+    const { portfolioHistory } = props
+    const lastPortfolioMetrics = last(portfolioHistory)
+    const [portfolioPeriod /*setPortfolioPeriod*/] = useState<Period>('allTime')
+
+    if (portfolioHistory.length === 0 || !lastPortfolioMetrics) {
+      return <div> No portfolio history data yet </div>
+    }
+
+    return (
+      <div>
+        <Row className="gap-8">
+          <div className="mb-4 w-full">
+            <Col>
+              <div className="text-sm text-gray-500">Portfolio value</div>
+              <div className="text-lg">
+                {formatMoney(
+                  lastPortfolioMetrics.balance +
+                    lastPortfolioMetrics.investmentValue
+                )}
+              </div>
+            </Col>
+          </div>
+          {
+            //TODO: enable day/week/monthly as data becomes available
+            /*<div>
+          <select
+            className="select select-bordered self-start"
+            value={portfolioPeriod}
+            onChange={(e) => setPortfolioPeriod(e.target.value as Period)}
+          >
+            <option value="daily">Daily</option>
+            <option value="weekly">Weekly</option>
+            <option value="monthly">Monthly</option>
+            <option value="allTime">All time</option>
+          </select>
+              </div>*/
```

Comment:
> One more idea would be to try to backfill the portfolio history for each user so that we could have the portfolio graph extend back to when the user signed up.
> 
> You'd have to interleave deposits from stripe, and profits from bets. Hmm I suppose it would be hard to know what the profit/loss of your bets were. I.e. you'd have to reconstruct the probability of every market over time. Theoretically you could do this by modeling all the bets of everyone over time...
> 
> Is there another way?

I don't think there's another way, although I agree that it's worth doing. I do expect it to be quite tricky for me to figure out all the relevant math (I think it would be different for each market type, and I don't really know the details of each one). Ideally one of you guys could help me break that down.